quiz/proxy/default.conf

```diff
@@ -13,4 +17,15 @@ server {
       proxy_set_header Connection "Upgrade";
       proxy_set_header Host $host;
     }
+
+    location /assets/ {
```

Comment:
Nie serwowałbym plików statycznych przez reverse proxy

Reply:
A jak inaczej mam podać statiki do django admina? Trochę nie mam innego pomysłu.

Reply:
Reverse proxy, służy do zabezpieczania wewnętrznej struktury serwera przez zewnętrznym światem, co pozwala nam serwować je z wielu miejsc naraz. Jedyne jaki widzę tutaj case, to wydajność, ale wtedy można zwiększyć ilość workerów nginx. Postaram się w przyszłym tygodniu wzgłębić tajniki nginx, do tej pory powinniśmy podjąć jednak decyzje, a później zmienić.

Reply:
@ToJestKrzysio Szukałam dzielnie innego sposobu na serwowanie staticów do django admina i jedyne co znalazłam to dodanie WhiteNoise, nałożenie cache i skompresowanie staticów dla django admina, ale dalej z użyciem proxy. Jakoś trzeba je podać, bo inaczej django admin wygląda smutno.